OrbitCore/LinuxTracingSession.cpp

```diff
@@ -23,6 +26,27 @@ void LinuxTracingSession::RecordHashedCallstack(
   hashed_callstack_buffer_.push_back(std::move(hashed_call_stack));
 }
 
+void LinuxTracingSession::SetStringManager(
+    std::shared_ptr<StringManager>& string_manager) {
```

Comment:
Pass shared_ptr by value or, if you are sure, by const reference.

Reply:
Done